ansible/roles/rabbitmq/templates/definitions.json.j2

```diff
@@ -16,13 +16,13 @@
     {"user": "{{ murano_agent_rabbitmq_user }}", "vhost": "{{ murano_agent_rabbitmq_vhost }}", "configure": ".*", "write": ".*", "read": ".*"}
     {% endif %}
   ],
-{% if rabbitmq_remove_ha_all_policy | bool %}
```

Comment:
We need to maintain backwards compatibility here. I think that means somthing like this:

```
if om_enable_rabbitmq_high_availability | bool
...
elif rabbitmq_remove_ha_all_policy | bool
...
else
...
```